user-allocations/.env.sample

```diff
@@ -0,0 +1,2 @@
+ETHERSCAN_API_KEY=''
```

Comment:
Shouldn't we be able to reuse the existing env file? else the user has to maintain 2 different env files

Reply:
yes